demisto_sdk/commands/common/constants.py

```diff
@@ -394,6 +401,9 @@ class FileType(Enum):
 
 PACKS_LAYOUTS_CONTAINER_JSON_REGEX = fr'{PACKS_LAYOUTS_DIR_REGEX}\/layoutscontainer([^/]+)\.json'
 
+PACKS_PRE_PROCESS_RULES_DIR_REGEX = fr'{PACK_DIR_REGEX}\/{PRE_PROCESS_RULES_DIR}'
```

Comment:
no need to escape a slash```suggestion
PACKS_PRE_PROCESS_RULES_DIR_REGEX = fr'{PACK_DIR_REGEX}/{PRE_PROCESS_RULES_DIR}'
```

Reply:
https://github.com/demisto/demisto-sdk/commit/12eaa1a394438b434ede91111fdb07add9f2e8c7